tests/unit_node/crypto/crypto_hash_test.ts

```diff
@@ -2,7 +2,7 @@
 import { createHash, createHmac, getHashes, hash } from "node:crypto";
 import { Buffer } from "node:buffer";
 import { Readable } from "node:stream";
-import { assert, assertEquals } from "@std/assert";
+import { assert, assertEquals } from "jsr:@std/assert";
```

Comment:
```suggestion
import { assert, assertEquals } from "@std/assert";
```
I needed to add jsr prefix to run the test manually.